drivers/wifi/esp32/Kconfig.esp32

```diff
@@ -0,0 +1,226 @@
+	  size of the cached TX queue.
+
+config ESP32_WIFI_DYNAMIC_TX_BUFFER_NUM
+	int "Max number of WiFi dynamic TX buffers"
+	depends on ESP32_WIFI_DYNAMIC_TX_BUFFER
+	range 1 128
+	default 32
+	help
+	  Set the number of WiFi dynamic TX buffers. The size of each
+	  dynamic TXbuffer is not fixed, it depends on the size of each
+	  transmitted data frame.
+
+	  For each transmitted frame from the higher layer TCP/IP stack,
+	  the WiFi driver makes a copy of it in a TX buffer. For some applications,
+	  especially UDP applications, the upper layer can deliver frames faster
+	  than WiFi layer can transmit. In these cases, we may run out of TX
+	  buffers.
+
+config ESP32_WIFI_CSI_ENABLED
+	bool "WiFi CSI(Channel State Information)"
+	help
+	  Select this option to enable CSI(Channel State Information) feature.
+	  CSI takes about CONFIG_ESP32_WIFI_STATIC_RX_BUFFER_NUM KB of RAM.
+	  If CSI is not used, it is better to disable this feature in order
+	  to save memory.
+
+config ESP32_WIFI_AMPDU_TX_ENABLED
+	bool "WiFi AMPDU TX"
+	default y
+	help
+	  Select this option to enable AMPDU TX feature. It improves transmission
+	  error checking and overall network performance with the cost of processing
+	  speed. Helpful when the device is operating in crowded wireless area.
+
+config ESP32_WIFI_TX_BA_WIN
+	int "WiFi AMPDU TX BA window size"
+	depends on ESP32_WIFI_AMPDU_TX_ENABLED
+	range 2 32
+	default 6
+	help
+	  Set the size of WiFi Block Ack TX window. Generally a bigger value means
+	  higher throughput but more memory. Most of time we should NOT change the
+	  default value unless special reason, e.g. test the maximum
+	  UDP TX throughput with iperf etc. For iperf test in shieldbox,
+	  the recommended value is 9~12.
+
+config ESP32_WIFI_AMPDU_RX_ENABLED
+	bool "WiFi AMPDU RX"
+	default y
+	help
+	  Select this option to enable AMPDU RX feature. It improves transmission
+	  error checking and overall network performance with the cost of processing
+	  speed. Helpful when the device is operating in crowded wireless area.
+
+config ESP32_WIFI_RX_BA_WIN
+	int "WiFi AMPDU RX BA window size"
+	depends on ESP32_WIFI_AMPDU_RX_ENABLED
+	range 2 32
+	default 6
+	help
+	  Set the size of WiFi Block Ack RX window. Generally a bigger value means
+	  higher throughput and better compatibility but more memory. Most of time
+	  we should NOT change the default value unless special reason,
+	  e.g. test the maximum UDP RX throughput with iperf etc. For iperf test in
+	  shieldbox, the recommended value is 9~12. If PSRAM is used and WiFi memory
+	  is preferred to allocat in PSRAM first, the default and minimum value
+	  should be 16 to achieve better throughput and compatibility with both
+	  stations and APs.
+
+choice ESP32_WIFI_TASK_CORE_ID
+	prompt "WiFi Task Core ID"
+	default ESP32_WIFI_TASK_PINNED_TO_CORE_0
+	help
+	  Pinned WiFi task to core 0 or core 1.
+
+	config ESP32_WIFI_TASK_PINNED_TO_CORE_0
+		bool "Core 0"
```

Comment:
Should there be second option for core 1 here?

Reply:
Well, no, running on core 1 is not supported yet.
I'll change the `help` comment in there to make in only available at core 0.